charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-node-windows.yaml

```diff
@@ -65,11 +65,11 @@ spec:
               mountPath: C:\registration
           resources:
             limits:
-              cpu: 2
-              memory: 2Gi
+              cpu: 1
+              memory: 1Gi
             requests:
-              cpu: 10m
-              memory: 20Mi
```

Comment:
`memory: 512Mi` could be too big, would waste resource, I think we could keep the original value